src/ProgressBar/Progress.less

```diff
@@ -5,6 +5,8 @@
   height: 100%;
   width: 100%;
   position: absolute;
+  transition: width 0.5s ease-out 0.25s;
+  -webkit-transition: width 0.5s ease-out 0.25s;
```

Comment:
Are the transitions too quick with these timing variables?
https://github.com/Clever/components/blob/c257f5b1c5cf7fdfe9e8a5f25a28c305829c6107/src/less/animations.less#L6-L10

Reply:
Oh cool I didn't realize these were already defined. 0.5s is 500ms so what I had set is actually slower than everything else! I'll probably do @timingSlowly for the transition duration and @timingPrompty for the delay.